remoteworkitem/remoteworkitem.go

```diff
@@ -58,6 +58,7 @@ const (
 	remoteCreatorProfileURL   = "system.creator.profile_url"
 	RemoteAssigneeLogins      = "system.assignees.login"
 	RemoteAssigneeProfileURLs = "system.assignees.profile_url"
+	remoteItemURL             = workitem.SystemRemoteItemURL
```

Comment:
`GithubID` on line no. 23 should be renamed as `GithubURL` (and similarly for Jira) -> https://github.com/fabric8-services/fabric8-wit/pull/2389/files#diff-b9d69c203555944a118b1e5b55713cc1R23

Reply:
There is no "id" in Github JSON output.